Makefile

```diff
@@ -43,7 +43,7 @@ endif
 BUILDMNT = /go/src/$(GOTARGET)
 # The version here should match the version of go configured in
 # .travis.yml
-BUILD_IMAGE ?= golang:1.14.4
+BUILD_IMAGE ?= golang:1.14.6
```

Comment:
maybe it's worth it using `golang:1.14` to always use the latest patch release?

Reply:
That wouldn't use the latest release if we already have golang:1.14 locally. I'd prefer to keep this explicit.

Reply:
> I'd prefer to keep this explicit

+1